### PR TITLE
feat(tokens): add borders, shadows

### PR DESCRIPTION
**Pull request by kamleshchandnani**

### CODE REVIEW

File: packages/blade/tokens/theme/lightTheme.ts [hunk 1]
```diff
@@ -1,340 +1,410 @@
...
+        highContrast: {
+          onLight: globalColors.neutral.blueGrayLight[0],
+          onDark: globalColors.neutral.blueGrayDark[0],
+        },
+        lowContrast: {
+          onLight: globalColors.chromatic.cider[600],
+          onDark: globalColors.chromatic.cider[600],
+        },
+      },
+      infromation: {
```
Comment (dhruvdutt) [inline packages/blade/tokens/theme/lightTheme.ts:216]:
```suggestion
      information: {
```

File: packages/blade/tokens/theme/neoTheme.ts [hunk 2]
```diff
@@ -1,340 +1,410 @@
...
+        },
+        lowContrast: {
           onLight: globalColors.chromatic.cider[600],
           onDark: globalColors.chromatic.cider[600],
         },
       },
-      gray: {
-        300: {
-          onLight: globalColors.neutral.navyGrayLight[50],
+      infromation: {
```
Comment (dhruvdutt) [inline packages/blade/tokens/theme/neoTheme.ts:216]:
```suggestion
      information: {
```

File: packages/blade/tokens/global/border.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  max: number;
+  round: string;
+}>;
+
+type BorderWidth = Readonly<{
+  none: number;
+  thin: number;
+}>;
+
+type Border = Readonly<{ radius: BorderRadius; width: BorderWidth }>;
```
Comment (dhruvdutt) [inline packages/blade/tokens/global/border.ts:15]: Please break this down into a new line.

File: packages/blade/tokens/global/border.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+const border: Border = {
+  radius: {
+    none: 0,
+    small: 2,
+    medium: 4,
+    large: 8,
+    max: 9999,
+    round: '50%', // this needs to be in % but need to figure out how will we store unitless things
```
Comment (dhruvdutt) [inline packages/blade/tokens/global/border.ts:24]: Having a different type only for this case would cause issues in usage. 😞

Comment (kamleshchandnani) [reply packages/blade/tokens/global/border.ts:24]: yes but we'll need to keep these exceptions as they'll come for such cases.

**Pull request merged**